closure/goog/ui/container.js

```diff
@@ -838,7 +838,7 @@ goog.ui.Container.prototype.registerChildId_ = function(child) {
 /**
  * Adds the specified control as the last child of this container.  See
  * {@link goog.ui.Container#addChildAt} for detailed semantics.
- * @param {goog.ui.Component} child The new child control.
+ * @param {goog.ui.Control} child The new child control.
```

Comment:
This actually has to stay a Component as it is overriding parent class's addChild()